src/components/core/tooltip/tooltip/tooltip-content.tsx

```diff
@@ -0,0 +1,155 @@
+import { HTMLAttributes, ReactElement } from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import { keyframes } from '@stitches/react';
+import useTheme from '@core/theme-context/use-theme';
+import { styled } from 'Styles/stitches.config';
+
+export type TTooltipContentProps = HTMLAttributes<HTMLDivElement> & {
+    children: ReactElement;
```

Comment:
> ReactElement is the type for elements in React, either created via JSX or React. createElement. ReactNode is wider, it can be text, number, boolean, null, undefined, a portal, a ReactElement, or an array of ReactNodes. It represents anything that React can render.

@vinu-deriv what do you think about using `ReactNode` instead of `ReactElement`?